Added velbus config flow documentation (#9955)
home-assistant/home-assistant.github.io: source/_components/velbus.markdown

---

> layout: page

Reviewer: We don't use this anymore, please remove this line.

title: "Velbus"
description: "Access and control your Velbus devices."

> date: 2017-06-17 16.58

Reviewer: We don't use this anymore, please remove this line.

> sidebar: true

Reviewer: We don't use this anymore, please remove this line.

> comments: false

Reviewer: We don't use this anymore, please remove this line.

> sharing: true

Reviewer: We don't use this anymore, please remove this line.

> footer: true

Reviewer: We don't use this anymore, please remove this line.

## Configuration

There are 2 options in configuring the velbus integration:

- Via the Home Assistant user interface where it will let you enter the port string to connect to the Velbus bus.
- Via the Home Assistant `configuration.yaml` file.

```yaml
# Example configuration.yaml entry for a USB or serial interface
velbus:
  port: 'PORT_STRING'
```

## Port Sstring

> The porti string used in the user interface or the configuration file can have 2 formats:

Reviewer 2: The port string

Reviewer: Addressed

## Configuration

> There are 2 options in configuring the velbus component:

Reviewer: ```suggestion
There are 2 options in configuring the Velbus integration:
```

> - Via the Home Assistant user interface where it will let you enter the portstring to connect to the velbus bus.

Reviewer: ```suggestion
- Via the Home Assistant user interface where it will let you enter the port string to connect to the Velbus bus.
```

- Via the Home Assistant `configuration.yaml` file.

```yaml
# Example configuration.yaml entry for a USB or serial interface
velbus:

>   port: '<portstring>'

Reviewer: ```suggestion
  port: 'PORT_STRING'
```

```

> ## Port String

Reviewer: ```suggestion
## Port string
```

> The portstring used in the user interface or the configuration file can have 2 formats:

Reviewer: ```suggestion
The port string used in the user interface or the configuration file can have 2 formats:
```
